test/katex-spec.js

```diff
@@ -1116,6 +1116,31 @@ describe("A markup generator", function() {
     });
 });
 
+describe("A parse tree generator", function() {
+    it("generates a tree", function() {
+        var tree = katex.__parse("\\sigma^2");
+        expect(JSON.stringify(tree)).toEqual(JSON.stringify([
```

Comment:
Surprisingly just doing `expect(tree).toEqual(...)` failed! Doing a string comparison seemed to work, though!